types/tabulator-tables/index.d.ts

```diff
@@ -1271,6 +1271,7 @@ You can pass an optional additional property with sorter, sorterParams that shou
     }
 
     type GroupValuesArg = any[][];
+    type GroupByArg = string | string[] | ((data: any) => any) | Array<((data: any) => any)>;
```

Comment:
Can you provide an example and test/docs of where this is used? I didn't see where you can pass in an array of functions

Reply:
Hello! It's not stated explicitly in the docs but you can validate it in the source code:

- https://github.com/olifolkerd/tabulator/blob/master/src/js/modules/group_rows.js#L819-L821 - here a single value is coerced into an array.

- https://github.com/olifolkerd/tabulator/blob/master/src/js/modules/group_rows.js#L826 - a couple lines after that there's a check if a function was passed.

Reply:
It's actually a good point that the docs don't mention this, I'll create a request for the docs change too.

Reply:
Ok thanks, I wonder if that is legacy code or meant for the public API. I'm definitely good with the `string[] `  change